src/components/Home/SupportSummary.tsx

```diff
@@ -0,0 +1,142 @@
+import {Trans} from '@lingui/react'
+import {JobIcon} from 'components/ui/JobIcon'
+import NormalisedMessage from 'components/ui/NormalisedMessage'
+import {GameEdition} from 'data/EDITIONS'
+import {JobKey, JOBS, RoleKey, ROLES} from 'data/JOBS'
+import {patchSupported} from 'data/PATCHES'
+import {FALLBACK_KEY, PATCHES} from 'data/PATCHES/patches'
+import {AVAILABLE_MODULES} from 'parser/AVAILABLE_MODULES'
+import React, {Component, Fragment} from 'react'
+import {Message} from 'semantic-ui-react'
+import styles from './SupportSummary.module.css'
+
+interface RoleData {
+	roleKey: RoleKey
+	jobKeys: JobKey[]
+}
+
+class SupportSummary extends Component {
+	private roleJobs: RoleData[] = []
+	override render() {
+		const maxPatch = [...Object.entries(PATCHES)].map(patch => patch[0]).pop() ?? FALLBACK_KEY
```

Comment:
I know that the PATCHES object is chronologically ordered for the sake of convenience, but this fact isn't necessarily guaranteed by the data type. Maybe something like:

```suggestion
		const maxPatch = Object.entries(PATCHES)
			.map(patch => ({name: patch[0], date: patch[1].date[GameEdition.GLOBAL]}))
			.sort((a, b) => b.date - a.date)[0].name
```